packages/client/utils/retroGroup/updateClonePosition.ts

```diff
@@ -25,22 +52,56 @@ const getTransition = (isClipped: boolean, timeRemaining: number) => {
 }
 
 export const getDroppingStyles = (
-  targetEl: HTMLDivElement,
+  element: HTMLDivElement,
   bbox: ClientRect,
   maxTop: number,
-  timeRemaining: number
+  timeRemaining: number,
+  targetId?: string | null,
+  isClose?: boolean,
+  lastZIndex?: number
 ) => {
   const spotlightEl = document.getElementById('spotlight')
-  const showAboveSpotlight = spotlightEl ? spotlightEl.contains(targetEl) : false
+  const isTargetInSpotlight = targetId
+    ? !!(spotlightEl && document.querySelector(`div[${DragAttribute.DROPPABLE}='${targetId}']`))
+    : false
+  const isInSpotlight = spotlightEl && spotlightEl.contains(element)
+  const showAboveSpotlight = isInSpotlight || isTargetInSpotlight
   const {top, left} = bbox
-  const minTop = getMinTop(top, targetEl)
+  const minTop = getMinTop(top, element)
   const clippedTop = Math.min(Math.max(minTop, top), maxTop - bbox.height)
   const isClipped = clippedTop !== top
+
+  const fadeInAnimation = `${reflectionSpotlightFadeIn.toString()} 0.5s linear 0s forwards`
+  const fadeOutAnimation = `${reflectionSpotlightFadeOut.toString()} 0.5s linear 0s forwards`
+
+  const notInSpotlightSuggestions = spotlightEl && !isInSpotlight
+  const isCurrentlyBehindSpotlight = lastZIndex !== ZIndex.REFLECTION_IN_FLIGHT_SPOTLIGHT
+  const isFadingIn =
+    // Spotlight is open and the element is not a part of the spotlight suggestions
```

Comment:
The original issue just used the terminology "source" for the reflection group at the top of the Spotlight and "results" for all of the similar Spotlight groups suggested at the bottom. I've been guilty of introducing new terminology like "Spotlight groups", but I think we should stick with just "source" and "results" wherever possible. 

So, I'd prefer to avoid using "Spotlight suggestions" if possible. It looks like we could just say "not a part of the spotlight" here.